### ISSUE TITLE
MNIST Training Enquiries

### ISSUE THREAD

**Issue by easonoob** [Tim-Li/cuTN-QSVM#1]
Great work! It is astonishing to hear that 784 qubits can be simulated on classical computers. In the paper, it is mentioned that 95% accuracy is achieved on the MNIST dataset with cuTN-QSVM, which is higher than classical training and testing accuracy. I am wondering how do you update the gate angles during training? Do you calculate the gradients and do backpropagation? Is it being done on cuTensorNet too? Or other methods are used instead of gradient descent? Thank you!
